docs/troubleshooting.md

```diff
@@ -217,3 +217,81 @@ The following is a simple example of this occurring:
 [INFO    2022/10/13 14:25:13]
 [FATAL   2022/10/13 14:25:13] Failed to initialize collector kernel components.
 ```
+
+## Troubleshooting using performance counters
+
+The collector publishes some performance counters that can be used to investigate runtime issues.
+
+The runtime values are exposed via the Prometheus endpoint `/metrics` and can be accessed on port 9090.
+
+### CollectorStats timers -> rox_collector_timers
```

Comment:
Since this docs are intended to help interpreting the metrics api output, probably worth mentioning that those are presented in triplets '<name>_events', '<name>_us_total', '<name>_us_avg'.

Reply:
A new little paragraph was added